packages/devtools_app/lib/src/ui/analytics.dart

```diff
@@ -154,17 +158,181 @@ class GtagExceptionDevTools extends GtagException {
   external String get ide_launched;
 }
 
+/// Request Flutter tool stored value enabled (GA enabled) stored in ~\.flutter.
+/// Return bool or null, null implies Flutter Tool has never been run.
+Future<bool> get isFlutterGAEnabled async {
+  bool enabled = false;
+
+  var resp =
+      await HttpRequest.request('/api/getFlutterGAEnabled', method: 'POST');
+  if (resp.status == HttpStatus.ok) {
+    // Return value 'null' implies Flutter tool has never been run return null.
+    if (resp.responseText == 'null') return null;
+
+    enabled = resp.responseText == 'true';
+  } else {
+    log('HttpRequest api/getFlutterGAEnabled failed status = ${resp.status}',
+        LogLevel.error);
+  }
+  return enabled;
+}
+
+/// Request Flutter tool stored clientID (GA enabled) stored in ~\.flutter.
+/// Return clientId as a String or null, null implies Flutter Tool has never
+/// been run.
+Future<String> flutterGAClientID() async {
+  String clientId = '';
+
+  var resp =
+      await HttpRequest.request('/api/getFlutterGAClientId', method: 'POST');
+  if (resp.status == HttpStatus.ok) {
+    // Return value 'null' implies Flutter tool has never been run return null.
+    if (resp.responseText == 'null') return null;
+
+    clientId = resp.responseText;
+  } else {
+    log('HttpRequest api/getFlutterGAClientId failed status = ${resp.status}',
+        LogLevel.error);
+  }
+  return clientId;
+}
+
+/// Request all .devtools properties resets to their default values.
+void resetDevTools() async {
+  final resp = await HttpRequest.request('api/resetDevTools', method: 'POST');
+  if (resp.status == HttpStatus.ok) {
+    assert(resp.responseText == 'true');
+  } else {
+    log(
+      'HttpRequest api/resetDevTools failed status = ${resp.status}',
+      LogLevel.error,
+    );
+  }
+}
+
+/// Request DevTools stored value firstRun (GA dialog) stored in ~\.devtools.
+Future<bool> get isFirstRun async {
+  bool firstRun = false;
+
+  var resp =
+      await HttpRequest.request('api/getDevToolsFirstRun', method: 'POST');
+  if (resp.status == HttpStatus.ok) {
+    firstRun = resp.responseText == 'true';
+  } else {
+    log('HttpRequest api/getDevToolsFirstRun failed status = ${resp.status}',
+        LogLevel.error);
+  }
+  return firstRun;
+}
+
+bool _gaEnabled;
+
+bool gaEnabled() => _gaEnabled;
+
+/// Request DevTools stored value enabled (GA enabled) stored in ~\.devtools.
+Future<bool> get isEnabled async {
+  if (_gaEnabled != null) return _gaEnabled;
+
+  bool enabled = false;
+
+  var resp =
+      await HttpRequest.request('/api/getDevToolsEnabled', method: 'POST');
+  if (resp.status == HttpStatus.ok) {
+    enabled = resp.responseText == 'true';
+  } else {
+    log('HttpRequest api/getDevToolsEnabled failed status = ${resp.status}',
+        LogLevel.error);
+  }
+
+  _gaEnabled = enabled;
+
+  return enabled;
+}
```

Comment:
same here:
1. dart doc - put enabled in backticks
2. create const string for api 
3. trailing comma after LogLevel.error

For this method and the rest below

Reply:
Already done.